<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtKalmanFilterExperimentDialog</class>
 <widget class="QDialog" name="QtKalmanFilterExperimentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KalmanFilterer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Experiment parameters</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="button_load">
       <property name="text">
        <string>Load from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_save">
       <property name="text">
        <string>Save as</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="box_save">
       <item>
        <property name="text">
         <string>DokuWiki</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HTML</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_n_timesteps">
       <property name="text">
        <string>Timesteps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_n_timesteps">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000000</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_n_states">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>States</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_add_state">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_remove_state">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
